tests/scripts/dind-cluster.sh

```diff
@@ -103,6 +105,7 @@ function dind::set-build-volume-args {
   if [ -n "${KUBEADM_DIND_LOCAL:-}" ]; then
     build_volume_args=(-v "$PWD:/go/src/k8s.io/kubernetes")
   else
+    # shellcheck disable=SC1091,SC2030 # source=somefile, var modification is local
```

Comment:
Use `# source=build/common.sh` instead of disabling SC1091.

SC2030 seems okay to disable, but comment it more clearly. `we want KUBE_ROOT=$PWD to only be in the subshell` more clearly indicates we know what the problem is and we intend the behavior